### ISSUE TITLE
Update Readme to reflect case rules

### ISSUE THREAD

**Issue by bondjimbond** [mnylc/islandora_multi_importer#112]
Somewhere, the column headings from the CSV are automatically transformed to lowercase. If you don't know this, and you write a Twig template with uppercase in your data, your template will not recognize the data because it expects, e.g. `First_Name`, while that value has been transformed into `first_name`.

README should make mention of this quirk and state that CSV column headings must all be lowercase, or that your Twig templates must be written with the understanding that column headings are transformed to lowercase.

**Comment by DiegoPino** [comment]
@bondjimbond true. That is something i want to document for Archipelago too. Twig properties (as in PHP variables) are always case insensitive and because of the limitations of how PHP variables can be normalized internally (as good as i could), so we thought it was better to normalize also to lower case. Will make sure to add an entry to the doc. Thanks!
